TSC-Charter.md

```diff
@@ -36,16 +36,27 @@ approved by the Board.
 
 ## Section 4. Establishment of the TSC.
 
-TSC memberships are not time-limited. There is no maximum size of the TSC.
-The size is expected to vary in order to ensure adequate coverage of important
-areas of expertise, balanced with the ability to make decisions efficiently.
-The TSC must have at least three members.
+TSC memberships are for one year terms that must be recertified. Motions for
+recertification are automatic and require a vote. Members are recertified if a
+simple majority of TSC members vote in favor of recertifying the individual as
+a TSC member. Such members will continue through the next year's term.
+Individuals failing recertification may return to the TSC after no less than
+six months following normal TSC motion.
```

Comment:
A tie would be less than simple majority so the individual would not be recertified. 

No, their vote for themselves wouldn't count. I can clarify that.